src/uipath_langchain/_cli/_runtime/_runtime.py

```diff
@@ -475,6 +482,25 @@ def __init__(
         self.resolver = LangGraphJsonResolver(entrypoint=entrypoint)
         super().__init__(context, self.resolver)
 
+    @cached_property
+    @override
+    def get_entrypoint(self) -> Entrypoint:
+        """Get entrypoint for this LangGraph runtime."""
+        with concurrent.futures.ThreadPoolExecutor(max_workers=1) as executor:
```

Comment:
I think we can just make this async in the base library
```python
async def get_entrypoint(self) -> Entrypoint:
```

Also it looks like the resolved graph is already cached in `LangGraphJsonResolver` (so `@cached_property` doesn't bring much value here)